cmd/cells/cells.go

```diff
@@ -35,10 +35,10 @@ import (
 	"io"
```

Comment:
line 31: Shouldn't you also rename 'package runtime' to 'package cells'?

Reply:
yes :)